main.go

```diff
@@ -34,7 +38,7 @@ func main() {
 		downstream string
 	)
 
-	flag.StringVar(&upstream, "upstream", "https://github.com/okkur/gitor.git", "specifies upstream")
+	flag.StringVar(&upstream, "upstream", upstream, "specifies upstream")
```

Comment:
That wasn't actually hardcoded. That was the default being used, if no flag is provided. So I think we can actually keep the default for now.